packages/account/src/Components/self-exclusion/__tests__/self-exclusion-article.spec.js

```diff
@@ -23,7 +23,7 @@ describe('<SelfExclusionArticle />', () => {
     const eu_item =
         /These trading limits and self-exclusion help you control the amount of money and time you spend on Deriv.com and exercise/i;
     const non_eu_item =
-        /These self-exclusion limits help you control the amount of money and time you spend trading on DTrader, DBot, and SmartTrader. The limits you set here will help you exercise/i;
+        /These self-exclusion limits help you control the amount of money and time you spend trading on DTrader, DBot, Binary Bot and SmartTrader. The limits you set here will help you exercise/i;
```

Comment:
I think you can remove the "/ /i" if you want to be specific. If not i think we can use lowercase

Reply:
thanks, sure, I will fix it. 
But in this case I also should check all tests connected with self-exclusion. The card is not related to tests.